Set/Models/GameModel.swift

```diff
@@ -122,15 +126,37 @@ struct GameModel {
         }
     }
     
-    mutating private func getBonus(for player: Player) {
+    private mutating func isFasterThanPreviousMatch() -> Bool {
+        let matchDate = Date.now
+        let matchTimeInterval = matchDate.timeIntervalSince(previousMatchDate ?? startGameDate)
```

Comment:
This part of code (function) should not modify somehow any values, it must do only what it means, return bool, please remember single responsibility principle